viz.Rmd

```diff
@@ -343,12 +538,14 @@ can_lang |>
 To answer our question, we will need to adjust the scale of the x and y axes so that they are on a log scale. Doing this can help make patterns in the data more interpretable.
 
 ```{r 03-mother-tongue-vs-most-at-home-scale, fig.width=5.75, fig.height=5, warning=FALSE, fig.cap = "Scatter plot of number of Canadians reporting a language as their mother tongue vs the primary language at home with log adjusted x and y axes"}
+library(scales)
+
 ggplot(can_lang, aes(x = most_at_home, y = mother_tongue)) +
   geom_point() +
-  xlab("Language spoken most at home \n (number of Canadian residents)") +
-  ylab("Mother tongue \n (number of Canadian residents)") +
-  scale_x_log10(labels = scales::comma) +
-  scale_y_log10(labels = scales::comma)
+  labs(x = "Language spoken most at home \n (number of Canadian residents)",
+       y = "Mother tongue \n (number of Canadian residents)") +
+  scale_x_log10(labels = comma) +
+  scale_y_log10(labels = comma)
 ```
 > Here we are adjusting the scale of the x and y axes on a $log$ scale with base 10. 
 > Notice that $log_{10}(10) = 1$, $log_{10}(100) = 2$, and $log_{10}(1000) = 3$. 
```

Comment:
@leem44 - I tried this, for relationship direction, strength and linearity. What do you think?

```
When we see two variables moving in the same direction 
(as x increases, y increases), we call this a *positive relationship*.
If instead, we saw two variables moving in the opposite direction 
(as x increases, y decreases), we would call this a *negative relationship*.
Another possibility here, is to observe no relationship. 
This is when y neither nor decreases across the values of x.
Because the points are fairly close together, 
we can say that the relationship is strong. 
If instead, we saw that the points were spread much further apart, 
we would say that the relationship was weak.
Because drawing a straight line through these points 
would fit the pattern we observe quite well, we say that it's linear.
If instead, drawing a straight line through these points 
did not fit the observed pattern of the data, 
we would say that it is non-linear.
```

Reply:
missing word: This is when y neither **increases** nor decreases...
I think we should add to the sentence: "Because the points in our graph are fairly close together," -- since we are going from talking about the general case to this specific case